src/GlassCat/generate.jl

```diff
@@ -15,7 +15,11 @@ Each catalog file is a module representing a distinct glass catalog (e.g. NIKON,
 AGF files in `sourcedir`. These are then included and exported in `mainfile`.
 """
 function generate_jls(
-    sourcenames::Vector{<:AbstractString}, mainfile::AbstractString, jldir::AbstractString, sourcedir::AbstractString
+    sourcenames::Vector{<:AbstractString},
+    mainfile::AbstractString,
+    jldir::AbstractString,
+    sourcedir::AbstractString;
+    glasstype::AbstractString = "AGF"
```

Comment:
`glasstype` arg adds support for non-AGF .jl file generation.

Reply:
Is this primarily for testing? I am right in thinking this code only supports generation from AGF files so would always write AGF glass types?

Reply:
Good point. It's probably clearer and safer just to have something like a `test` bool then.